config/routes.rb

```diff
@@ -95,10 +95,6 @@
     match '/:person_id/settings/payments/paypal_account/show' => 'paypal_accounts#show', :as => :show_paypal_account_settings_payment
     match '/:person_id/settings/payments/paypal_account/create' => 'paypal_accounts#create', :as => :create_paypal_account_settings_payment
 
-    scope module: "atom", constraints: { format: "atom" } do
```

Comment:
This generates route: ```<locale>/listings.atom?<params>``` which is duplicate route with what ```resources :listings``` creates. Moreover, the first one does not limit the format to "atom" because of how rails works. 

This fixes some requests to urls ```<locale>/listings``` that have generated now a TemplateNotFoundError.